src/main/java/frc/robot/maps/subsystems/patterns/ElevatorFill.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.maps.subsystems.patterns;
+
+import com.chopshop166.chopshoplib.leds.AnimatedPattern;
+import com.chopshop166.chopshoplib.leds.SegmentBuffer;
+
+import edu.wpi.first.networktables.DoubleSubscriber;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.util.Color;
+import frc.robot.maps.subsystems.ElevatorMap;
+import frc.robot.maps.subsystems.AlgaeDestageMap.Data;
+import frc.robot.subsystems.Elevator;
+
+public class ElevatorFill extends AnimatedPattern {
```

Comment:
At some point we should make this more generic and add the topic as a parameter to the constructor.